### PR TITLE
feat(examples): add Aurelia example with buttons, checkboxes and ripple

### PR DESCRIPTION
**Pull request by Thanood**
This PR adds an Aurelia app to the examples section, containing these components done with Aurelia:
- mdl-button
- mdl-checkbox
- mdl-ripple

It shows usage, binding properties and event usage.
The app is using aurelia-skeleton-webpack as its base.

### CODE REVIEW

**Comment by Thanood** [comment]
CLA signed,

File: examples/aurelia/custom_typings/fetch.d.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/// <reference path="../node_modules/@types/whatwg-fetch/index.d.ts" />
+interface URLSearchParams {}
+declare module "isomorphic-fetch" {
+  export = fetch;
+}
```
Comment (traviskaufman) [inline examples/aurelia/custom_typings/fetch.d.ts]: nit: add a newline at the EOF

**Comment by Thanood** [comment]
Rebased and changes applied.

According to #4837 the imports should look like this:
```javascript
const {MDLCheckbox} = require('mdl-checkbox');
```

And this is what worked. 😃

File: examples/aurelia/custom_typings/fetch.d.ts (hunk 1 above)
Comment (Thanood) [inline examples/aurelia/custom_typings/fetch.d.ts]: added newline

**Comment by traviskaufman** [comment]
_Excellent_ work again! That demo is pretty sweet. Glad to see that ripples hook up seamlessly to buttons 😄

**Comment by Thanood** [comment]
Thanks!
I'd also like to say thanks for MDLv2. Once I found the readme files it was very easy to integrate. 😉 
*Especially* ripples (attach and done). I love the new structure so far.

**Pull request merged**